package.json

```
@@ -4,7 +4,9 @@
 	"private": true,
 	"license": "agpl-3.0",
 	"workspaces": {
-		"packages": ["packages/*"]
+		"packages": [
+			"packages/*"
+		]
```

```suggestion
		"packages": ["packages/*"]
```
sorry. overzealous IDE.